### ISSUE TITLE
DateTime picker opens up twice and doesn't set value.

### ISSUE THREAD

**Issue by runedrune** [danvick/flutter_form_builder#121]
```
                      FormBuilderDateTimePicker(
                        firstDate: DateTime.now(),
                        attribute: "datetime",
                        decoration: InputDecoration(
                            border: OutlineInputBorder(),
                            labelText: S.of(context).addProgramProgramDateTime),
                        validators: [FormBuilderValidators.required()],
                      ),
```

Using this code I keep getting this:

![ezgif-2-b38eabe82fa4](https://user-images.githubusercontent.com/6131045/64010887-0be7f680-cb34-11e9-8a53-9c1d3665e594.gif)

```
[✓] Flutter (Channel stable, v1.7.8+hotfix.4, on Linux, locale en_US.UTF-8)
    • Flutter version 1.7.8+hotfix.4 at /home/fawad/Development/flutter
    • Framework revision 20e59316b8 (6 weeks ago), 2019-07-18 20:04:33 -0700
    • Engine revision fee001c93f
    • Dart version 2.4.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /home/fawad/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /home/fawad/Development/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Android Studio (version 3.5)
    • Android Studio at /home/fawad/Development/android-studio
    • Flutter plugin version 38.2.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.37.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.3.0

[✓] Connected device (1 available)
    • SM G975F • 10.42.0.163:5556 • android-arm64 • Android 9 (API 28)

• No issues found!

```

**Comment by sapphire008** [comment]
I have the same issue.

**Comment by danvick** [comment]
Hi @runedrune,
Could you kindly confirm that this issue is fixed - in v3.5.1

**Comment by runedrune** [comment]
Everything working perfectly. Thanks!
